spring-boot-project/spring-boot-autoconfigure/src/test/java/org/springframework/boot/autoconfigure/orm/jpa/HibernatePropertiesTests.java

```diff
@@ -135,6 +136,13 @@ void defaultDdlAutoIsNotInvokedIfHibernateSpecificPropertyIsSet() {
 				.run(assertDefaultDdlAutoNotInvoked("create"));
 	}
 
+	@Test
+	void defaultDdlAutoIsNotInvokedIfJpaDatabaseActionPropertyIsSet() {
+		this.contextRunner
+				.withPropertyValues("spring.jpa.properties.javax.persistence.schema-generation.database.action=create")
+				.run(assertDefaultDdlAutoNotInvoked("create"));
```

Comment:
I don't think this test is correct. The [spec](https://docs.oracle.com/javaee/7/tutorial/persistence-intro005.htm) states that `drop-and-create` is a valid value. If you'd rewrite this test with this, you'd have to assert that ddl auto was set with it, which is not a valid ddlAuto.

Rather, we should check the ddl auto isn't set (but the spec property is).